sdk/pom.xml

```diff
@@ -143,6 +143,14 @@
 			<scope>test</scope>
 		</dependency>
 
+		<!-- https://mvnrepository.com/artifact/javax.xml.bind/jaxb-api -->
```

Comment:
Note: should be automatically removed after merging with changes of https://github.com/ZencashOfficial/Sidechains-SDK/pull/68